juju/juju
# feat: add a controller serivce with ControllerModelUUID method

Add a controller service to handle retrival of core controller values that are related to the controller but do not fall into the category of controller config

Add a method to retrive the controllerModelUUID. Primarily to be used by the controlsocket worker in a future change.





## Checklist



~- [ ] [Integration tests](https://github.com/juju/juju/tree/main/tests), with comments saying what you're testing~

## QA steps
```
go test -v -tags=libsqlite3,dqlite ./domain/controller/... -check.v -check.f Suite
```


## Documentation changes

Added doc.go for new domain.
## Links

**Jira card:** https://warthogs.atlassian.net/browse/JUJU-6525
